docs/credstores.md

```diff
@@ -140,15 +227,13 @@ Permissions on existing directories will not be modified.
 
 ---
 
-<p align="center">
-
 :warning: **WARNING** :warning:
 
 **This storage mechanism is NOT secure!**
 
-**Secrets and credentials are stored in plaintext files _without any security_!<br/>
-Git Credential Manager Core takes no liability for the safety of these
-credentials.**
+**Secrets and credentials are stored in plaintext files _without any security_!**
+
+**Git Credential Manager Core takes no liability for the safety of these credentials.**
```

Comment:
This is an odd disclaimer. Consider replacing with "We strongly advise against using GCM Core in this configuration" or something like that.

Reply:
I'll drop this line altogether since the line immediately below already says that.